benchmarks/davies_et_al/doc/davies_et_al.md

```diff
@@ -6,14 +6,13 @@ All of the benchmarks presented so far take place in a Cartesian domain.
 Davies et al.&nbsp;describe a benchmark (in a paper that is currently still
 being written) for a 2D spherical Earth that is nondimensionalized such that
 
-<div class="table*">
-
+```{table}
+:name: tab:davies_et_al
 |                   |                                  |
 |:------------------|:---------------------------------|
 | $r_{\min}$ = 1.22 | $\left. T \right|_{r_{min}}$ = 1 |
 | $r_{\max}$ = 2.22 | $\left. T \right|_{r_{max}}$ = 0 |
```

Comment:
This table does not appear right in the built documentation: https://aspect-documentation--4944.org.readthedocs.build/en/4944/user/benchmarks/benchmarks/davies_et_al/doc/davies_et_al.html